pkg/server/image_pull.go

```diff
@@ -99,14 +99,20 @@ func (c *criContainerdService) PullImage(ctx context.Context, r *runtime.PullIma
 
 	// TODO(mikebrow): add truncIndex for image id
 	image, err := c.client.Pull(ctx, ref,
-		containerd.WithPullUnpack,
 		containerd.WithSchema1Conversion,
 		containerd.WithResolver(resolver),
-		containerd.WithPullSnapshotter(c.config.ContainerdConfig.Snapshotter),
 	)
 	if err != nil {
 		return nil, fmt.Errorf("failed to pull image %q: %v", ref, err)
 	}
+
+	// Do best effort unpack.
+	glog.V(4).Info("Unpack image %q", imageRef)
```

Comment:
/s/Unpack/First unpack image attempt/  ?

Reply:
We expect this to succeed in most cases. :) If this fails, it is an error to us.

The unpack before creation is just a fall back to handle the error.

Reply:
But if we handle an error do we want it reported?

Reply:
Hm, yeah, maybe warning makes more sense then.

Reply:
Done